### ISSUE TITLE
add explanation of throttle inter-node data replication set to true

### ISSUE THREAD

**Issue by lwandzura** [influxdata/docs-v2#4079]
Question from a customer from @shriramb380 :

 In the latest release of 1.9.7 we have an update "Throttle inter-node data replication, both incoming writes and hinted hand-off, when errors are encountered." 

Customer is asking, 1.  Is this communicated via /write response code - meaning customer should expect to see increased shard write errors (in a good way)? Or is the throttling communicated in a different way (in which case they might even expect to see fewer shard write errors)? or 2. They  expect to see increased volume in the HH queues if throttling kicks in?

Per @davidby-influx  from Slack thread:

When a write fails to a remote node (whether from HH or the initial remote write), the retry previously would not have been delayed. For instance, if allow-out-of-order-writes=true , if a remote write failed, it would be stored in the HH queue.  However, the next incoming remote write would be attempted as soon as it came it came in.

In the new version, when a remote write fails and allow-out-of-order-writes=true, the database will send data to the HH queue for a period before allowing incoming remote writes to be tried again.  This period starts quite short, and increases with each failed write.  So, the greater the number of successive failures to write, the longer a node will delay before retrying.
This same back-off strategy is used when emptying an HH queue.  When a node decides to try to pull items off the HH queue and resend them to the remote node, every successive failure will cause the node to wait longer before retrying.
TLDR; consecutive write failures cause increasing back-off times before retrying, in both emptying HH queues and in remote writes when allow-out-of-order-writes=true.

The result will could be temporarily larger HH queues, and more partial write errors if the consistency level is not ANY  when HH is written to, but fewer errors ("remote write failed") emptying HH queues, and much lower memory usage when a node is down (because remote writes, which keep data in memory while timing out, will be suppressed).

It is important to note that the back-off only occurs on the second failed write. So if the remote write failures (from the HH queue or directly incoming) are transient, the behavior and HH queue size will not differ in this version. Only if there are repeated write failures will the database slow down its retry rate, generating more "partial write" errors and fewer remote "write failed errors" and temporarily larger HH queues.

Let's update our docs with this information.

Docs to update:

https://docs.influxdata.com/enterprise_influxdb/v1.9/administration/configure/config-data-nodes/#allow-out-of-order-writes
